src/tools/dev_server/hot-reload-server.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import WebSocket from 'ws';
+import chokidar from 'chokidar';
+
+export class HotReloadServer {
```

Comment:
@piotrswigon ideally we'd add unit or integration tests for this server/logic in a later PR?